ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -38,7 +38,7 @@ def hardshrink(x, threshold=0.5, name=None):
 
 @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
 @to_ivy_arrays_and_back
-def hardswish(x, name=None):
+def hardswish(x, /, *, name=None):
```

Comment:
Kindly remove this change in the frontend. The signature here should be like the original function. 
Other than that we are ready to merge!